src/main/java/com/poiji/bind/mapping/PoijiHandler.java

```diff
@@ -185,6 +180,23 @@ private boolean setValue(Field field, int column, String content, Object ins) {
         return false;
     }
 
+    private Integer getFieldColumnFromExcelCellName(final ExcelCellName excelCellName) {
+        final String titleName = options.getCaseInsensitive()
+            ? excelCellName.value().toLowerCase()
+            : excelCellName.value();
+        if (excelCellName.delimeter().isEmpty()) {
+            return columnIndexPerTitle.get(titleName);
+        } else {
+            final String[] possibleTitles = titleName.split(excelCellName.delimeter());
+            for (final String possibleTitle : possibleTitles) {
```

Comment:
thank so much you for you effort @vaa25 ,  but this is the case I care about more. So we don't need a if-else statement + a for loop. Thats why it leads a performance for huge excel files.

Reply:
@ozlerhakan did you run ConcurrencyTest? What performance penalty did you catch for huge file?

Reply:
@ozlerhakan review again please.

Reply:
@ozlerhakan read performance increased for 30%